src/Data/List/Ext.agda

```diff
@@ -1,12 +1,66 @@
 {-# OPTIONS --safe #-}
 module Data.List.Ext where
 
-open import Agda.Builtin.List using (List; []; _∷_)
-open import Agda.Builtin.Maybe using (Maybe; nothing; just)
-open import Agda.Builtin.Nat using (Nat; zero; suc)
+open import Data.List using (List; [_]; _++_; map)
```

Comment:
It's generally recommended to use standard library (as opposed to built-ins) when possible.  Was there a reason for using built-ins here?